services/wallet/api.go

```diff
@@ -119,9 +125,27 @@ func (api *API) GetTokensBalancesForChainIDs(ctx context.Context, chainIDs []uin
 	return api.s.tokenManager.GetBalances(ctx, clients, accounts, addresses)
 }
 
-// GetBalanceHistory retrieves native token. Will be extended later to support token balance history
-func (api *API) GetBalanceHistory(ctx context.Context, chainID uint64, address common.Address, timeInterval transfer.BalanceHistoryTimeInterval) ([]transfer.BalanceState, error) {
-	return api.s.transferController.GetBalanceHistory(ctx, chainID, address, timeInterval)
+func (api *API) StartBalanceHistory(ctx context.Context) error {
+	api.s.history.StartBalanceHistory()
+	return nil
+}
+
+func (api *API) StopBalanceHistory(ctx context.Context) error {
+	api.s.history.Stop()
+	return nil
+}
```

Comment:
Do we need dedicated Start/Stop for the history, cannot we use the global Start/Stop Wallet?

Reply:
I don't want to start for mobile. However, it should be merged when it is implemented from their side.

Reply:
Mobile do not use the StartWallet function yet, it can be merged no?

Reply:
Then indeed. However, I would like to postpone it until the entire work is finished and integrated into the app.